wagtail/snippets/templates/wagtailsnippets/snippets/listing_buttons.html

```diff
@@ -0,0 +1,5 @@
+{% for button in buttons %}
+    {% if button.show %}
+        <li>{{ button|safe }}</li>
```

Comment:
Is using `|safe` here, safe?

Reply:
Yep, this was copied from https://github.com/wagtail/wagtail/blob/master/wagtail/admin/templates/wagtailadmin/pages/listing/_buttons.html